Source/Charts/Charts/ChartViewBase.swift

```diff
@@ -439,33 +444,37 @@ open class ChartViewBase: NSUIView, ChartDataProvider, AnimatorDelegate
     /// Highlights the value at the given x-value and y-value in the given DataSet.
     /// Provide -1 as the dataSetIndex to undo all highlighting.
     /// This method will call the delegate.
-    /// - parameter x: The x-value to highlight
-    /// - parameter y: The y-value to highlight. Supply `NaN` for "any"
-    /// - parameter dataSetIndex: The dataset index to search in
-    /// - parameter dataIndex: The data index to search in (only used in CombinedChartView currently)
+    ///
+    /// - Parameters:
+    ///   - x: The x-value to highlight
+    ///   - y: The y-value to highlight. Supply `NaN` for "any"
+    ///   - dataSetIndex: The dataset index to search in
+    ///   - dataIndex: The data index to search in (only used in CombinedChartView currently)
     @objc open func highlightValue(x: Double, y: Double, dataSetIndex: Int, dataIndex: Int = -1)
     {
         highlightValue(x: x, y: y, dataSetIndex: dataSetIndex, dataIndex: dataIndex, callDelegate: true)
     }
     
     /// Highlights any y-value at the given x-value in the given DataSet.
     /// Provide -1 as the dataSetIndex to undo all highlighting.
-    /// - parameter x: The x-value to highlight
-    /// - parameter dataSetIndex: The dataset index to search in
-    /// - parameter dataIndex: The data index to search in (only used in CombinedChartView currently)
-    /// - parameter callDelegate: Should the delegate be called for this change
+    /// - Parameters:
```

Comment:
why some ` /// - Parameters:` has `///` line above, while some like this don't have? Is this a mistake?